apps/src/scratch/scratch.js

```diff
@@ -14,7 +14,7 @@ import { singleton as studioApp } from '@cdo/apps/StudioApp';
 import ScratchView from './ScratchView';
 import { scratchDefaultProject } from './scratchDefaultProject';
 
-const Scratch = window.Scratch = window.Scratch || {};
+const Scratch = window.__TestInterface__ = {};
```

Comment:
Hah, I just realized my link above cut off the trailing "__".  There's also: https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=__TestInterface__&type=